nov-13-hardfork-spec.md

```diff
@@ -0,0 +1,38 @@
+# November 13th Bitcoin Cash Hardfork Technical Details
+
+Version 1.0, 2017-12-01
```

Comment:
let's hope we don't release it that late (date is wrong)

Reply:
fdhsafjklas...